testpilot/frontend/static-src/app/views/settings-menu-view.js

```diff
@@ -13,8 +13,8 @@ export default BaseView.extend({
 
                <div class="settings-menu no-display">
                  <ul>
-                   <li><a data-l10n-id="menuWiki" href="https://wiki.mozilla.org/Test_Pilot" target="_blank">Test Pilot Wiki</a></li>
-                   <li><a data-l10n-id="menuFileIssue" href="https://github.com/mozilla/testpilot/issues/new" target="_blank">File an Issue</a></li>
+                   <li><a data-l10n-id="menuWiki" data-hook="wiki" href="https://wiki.mozilla.org/Test_Pilot" target="_blank">Test Pilot Wiki</a></li>
```

Comment:
Using the `href` plus sending the callback to GA results in the page opening in both the new tab and the current tab. There are some example of turning external links into [`<button>` tags and handling the redirect in the view.](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/templates/landing-page.js#L111).

There's also [new code that will open links in new tabs when `data.newTab` is set](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/main.js#L68).